README.md

```diff
@@ -1,24 +1,26 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# ToDo List
 
-# Awesome Books
-
-> In this project I am using ES6 Modules on creating favorite books where we can save our favorit books!
+> In this project, we can create our daily todo lists
 
 ## Built With
 
-- Javascript
 - HTML
 - CSS
+- Javascript
+
+## Setup
+
+To get a local copy follow these simple example steps.
 
-## Live Demo
+1.- Clone the repository using this command:  
+`git@github.com:sediqullahbadakhsh/todo-list.git`
 
```

Comment:
- [ ] Kindly add instruction on installing dependencies(`npm install` or `npm i`) and running your project locally(`npm start)

Reply:
ok thanks, I will